packages/pagination/src/__tests__/hooks/useBreakpoint.hook.test.ts

```diff
@@ -1,3 +1,4 @@
+//NOTE: This has been moved to theme package. Please update references and remove this version when possible
```

Comment:
once this is merged we can create a ticket to remove this package specific version of useBreakpoint